src/System.Numerics.Tensors/System.Numerics.Tensors.sln

```diff
@@ -0,0 +1,28 @@
+﻿
```

Comment:
> FWIW this is what we do in CoreFx as well

That is true. Good point.

> I prefer a single solution for my development since I only care about this library. 

Fair enough :)